### ISSUE TITLE
Restructure buildpack: Create php-nginx buildpack 

### ISSUE THREAD

**Issue by sophiewigmore** [paketo-buildpacks/php#479]
## Context
Per [PHP RFC 0001](https://github.com/paketo-buildpacks/rfcs/blob/main/text/php/0001-restructure.md) we are restructuring the PHP language family of buildpacks to make them more modular, up-to-date, and as a result, maintainable.

Currently, the [PHP Web buildpack ](https://github.com/paketo-buildpacks/php-web) is responsible for configuring and running PHP applications across built-in PHP servers, HTTPD, NGINX, as well as supporting FPM process managers. The goal of this issue is to separate the concerns within PHP Web into individual buildpacks.

## Issue
This issue is to add a new buildpack, named `php-nginx`, which will be responsible for setting up NGINX to serve PHP applications. **This issue is blocked on https://github.com/paketo-buildpacks/php/issues/476.**

Logic to support the NGINX server setup currently lives within the [PHP Web buildpack ](https://github.com/paketo-buildpacks/php-web/blob/v0.1.3/features/nginx.go) and should be moved into this new buildpack.

- [ ] API: The buildpack should `provide none` and `require php`, `require php-fpm`, and `require nginx` at launch-time. 
- The buildpack will be responsible for generating an `nginx.conf` file
- The buildpack should contribute NGINX-specific FPM configurations to the location set up in in the PHP FPM buildpack. This configuration should at a minimum include setting the `www/listen` value to the same value as the FPM Socket in the NGINX config. (See [this example](https://github.com/paketo-buildpacks/php-web/blob/676fe1238863955f108ef086875692a6e2144e61/features/phpfpm.go#L62-L66) in PHP web, as well as [this example](https://github.com/paketo-buildpacks/php-web/blob/676fe1238863955f108ef086875692a6e2144e61/features/nginx.go#L48))
- It is also responsible for setting the start command of type `web` to start the NGINX server, as well as PHP FPM.
- In order for this buildpack to pass detection, users must declare their intention to use NGINX as their server of choice. Following https://github.com/paketo-buildpacks/php/issues/472, an option to accomplish this besides using a `buildpack.yml` should be enumerated to cover this case. 
 
- [ ] Packit: This buildpack should be written using the [packit library](https://github.com/paketo-buildpacks/packit).

- [ ] Buildpack API Version: This buildpack should be written using [Buildpack API 0.7](https://github.com/buildpacks/spec/releases/tag/buildpack%2Fv0.7)

- [ ] Add a `[[buildpack.licences]]` section to the `buildpack.toml`.

- [ ] README: Make sure a README is added that conveys context about the buildpack in a manner consistent with other Paketo buildpacks.

- [ ] Direct processes: Use direct processes and exec.d where applicable.

This issue can be transferred to the appropriate repository (paketo-buildpacks/php-nginx), once it has been created.

## Acceptance
1. I can successfully build a PHP app that uses an NGINX such as [the Paketo PHP NGINX sample app](https://github.com/paketo-buildpacks/samples/tree/main/php/nginx), with PHP Dist, the new PHP-FPM buildpack, the NGINX buildpack, this new PHP NGINX buildpack. During the build, I can see logs that show the app is run with the NGINX server and starts FPM. I can also run the container and curl the app and see `Powered By Paketo Buildpacks`.

**Comment by sophiewigmore** [comment]
CLosing in favour of https://github.com/paketo-buildpacks/php/issues/508 and https://github.com/paketo-buildpacks/php/issues/507
